news/_posts/2019-11-06-spark-3.0.0-preview.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: Preview release of Spark 3.0
+categories:
+- News
+tags: []
+status: publish
+type: post
+published: true
+meta:
+  _edit_last: '4'
+  _wpas_done_all: '1'
+---
+To enable wide-scale community testing of the upcoming Spark 3.0 release, the Apache Spark community has posted a <a href="https://archive.apache.org/dist/spark/spark-3.0.0-preview/">preview release of Spark 3.0</a>. This preview is <b>not a stable release in terms of either API or functionality</b>, but it is meant to give the community early access to try the code that will become Spark 3.0. If you would like to test the release, please download it, and send feedback using either the <a href="https://spark.apache.org/community.html">mailing lists</a> or <a href="https://issues.apache.org/jira/browse/SPARK/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel">JIRA</a>.
+
+The Spark issue tracker already contains a list of <a href="https://issues.apache.org/jira/browse/SPARK-26078?jql=statusCategory%20%3D%20done%20AND%20project%20%3D%2012315420%20AND%20fixVersion%20%3D%2012339177%20ORDER%20BY%20priority%20DESC%2C%20key%20ASC">features in 3.0</a> and <a href="https://issues.apache.org/jira/browse/SPARK-27471?jql=project%20%3D%2012315420%20AND%20%22Target%20Version%2Fs%22%20%20%3D%2012339177%20AND%20statusCategory%20in%20(%22To%20Do%22%2C%20%22In%20Progress%22)%20ORDER%20BY%20priority%20DESC%2C%20key%20ASC">known issues in the preview</a>.
```

Comment:
is this the right link?

Reply:
Since there is no `3.0.0-preview` version, I used version code for `3.0.0` instead.

Reply:
When I open these 2 links, it's just a single ticket, not a list.

Reply:
The links work OK for me; I get a list.
I wouldn't include the second link; they're not necessarily known issues nor even things that will be 3.0. (We need to proactively prune that list but I'm not sure most people have.)

Reply:
Thanks, updated.